geometry_msgs/Vector3 lla
float64 counter
bool under_water